digestparser/output.py

```diff
@@ -52,10 +51,10 @@ def docx_file_name(digest, digest_config=None):
     default_file_name_pattern = u'{author}_{msid:0>5}.docx'
     file_name_pattern = default_file_name_pattern
     if digest_config and 'output_file_name_pattern' in digest_config:
-        file_name_pattern = unicode_value(digest_config.get('output_file_name_pattern'))
+        file_name_pattern = str(digest_config.get('output_file_name_pattern'))
     # collect the values from the digest if present
     file_name = file_name_pattern.format(
-        author=utils.unicode_decode(digest.author),
+        author=digest.author,
```

Comment:
if you're going to return to this later did you want to put some `# TODO: python3 decode` type message here?

Reply:
I'm fairly confident `unicode_decode()`, which is removed as part of this PR, was only to account for `py27` content; for Python 3 it just returned the original string if it encountered an exception.

The only TODO in this library I noticed is on this line https://github.com/elifesciences/digest-parser/blob/develop/digestparser/zip.py#L39 where the string `.decode()` is still present.